src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -33,6 +33,7 @@
 #include <shogun/lib/SGVector.h>
 #include <shogun/mathematics/eigen3.h>
 #include <shogun/mathematics/linalg/LinalgBackendBase.h>
+#include <Eigen/Core>
```

Comment:
@karlnapf I don't have to include `<Eigen/Core>` to make it work on my machine, and the codes failed on travis even with the header. It seems that  `Eigen::UpLoType` isn't taken as a known enum class on travis and I am not sure why.